service/src/test/java/eu/europa/ec/fisheries/ers/activity/service/bean/ActivityMessageConsumerBeanTest.java

```diff
@@ -55,20 +48,20 @@ This file is part of the Integrated Fisheries Data Management (IFDM) Suite. The
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 import static org.mockito.Matchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
-import static org.powermock.api.mockito.PowerMockito.mockStatic;
 
 /**
  * Created by kovian on 17/07/2017.
  */
 @RunWith(PowerMockRunner.class)
-@PrepareForTest({MappedDiagnosticContext.class})
-@PowerMockIgnore({"javax.management.*"})
+//@PrepareForTest({MappedDiagnosticContext.class})
+//@PowerMockIgnore({"javax.management.*"})
 // TODO: Rewrite this test without powermock
```

Comment:
You could remove all of these three lines

Reply:
Oops